igor-web/src/main/groovy/com/netflix/spinnaker/igor/scm/gitlab/client/GitLabClient.groovy

```diff
@@ -0,0 +1,22 @@
+package com.netflix.spinnaker.igor.scm.gitlab.client
+
+import com.netflix.spinnaker.igor.scm.gitlab.client.model.CompareCommitsResponse
+import retrofit.http.GET
+import retrofit.http.Path
+import retrofit.http.QueryMap
+
+/**
+ * Interface for interacting with a Gitab REST API
+ * https://docs.gitlab.com/ce/api/README.html
+ */
+interface GitLabClient {
+
+    /**
+     * https://docs.gitlab.com/ce/api/repositories.html#compare-branches-tags-or-commits
+     */
+    @GET('/api/v4/projects/{projectKey}%2F{repositorySlug}/repository/compare')
```

Comment:
You can't use `/` instead of `%2F`?

Reply:
Not really, because Gitlab API expects a URL-encoded project id: 
```
id (required) - The ID or URL-encoded path of the project owned by the authenticated user
```
From https://docs.gitlab.com/ce/api/repositories.html#compare-branches-tags-or-commits

Reply:
OK 👍